### ISSUE TITLE
Spiceinit issues with ISIS cube DEMs with 7.0 RC1

### ISSUE THREAD

**Issue by volcanopele** [DOI-USGS/ISIS3#4805]
**ISIS version(s) affected**: 7.0.0 RC1

**Description**  
Spiceinit produces errors when using a cube DEM is used as the shape model. The error in the terminal looks like this:

`Error                    = "An unknown NAIF error has been encountered. The
                              short explanation provided by NAIF is
                              [SPICE(UNKNOWNFILARC)]. The Naif error is [The
                              file,
                              /HiRISE/Data/NAIF/molaDTM/molaMarsPlanetaryRadi-
                              us_HiRISE_v2.cub, has a unidentified file
                              architecture.  Check that this file is a
                              properly created binary SPICE kernel.]”`

**How to reproduce**  
The following example commands, on a CaSSIS framelet, cause this error on our system

`spiceinit from=MY36_018242_015_0_000_BLU.cub shape=user model=/HiRISE/Data/NAIF/molaDTM/molaMarsPlanetaryRadius_HiRISE_v2.cub attach=true spkpredict=true ckpredict=true` 

`spiceinit from=MY36_018242_015_0_000_BLU.cub attach=true spkpredict=true ckpredict=true`

In the CaSSIS geometry pipeline, we do use a custom MOLA DTM for the shape model but this error can be reproduced with just the default Mars shape model for ISIS: base/dems/molaMarsPlanetaryRadius0005.cub, as shown with the second command. I'll check to see if this is also an issue with CTX images, but I doubt this is camera-specific.

**Possible Solution**  
While the GUI version of spiceinit suggests that a cube can be used as a shape model as well as a DSK, it looks like spiceinit as run is only looking for a DSK kernel.

**Additional context**  
I'm performing testing of ISIS 7.0 RC1, installed on the UofArizona PIRL system on 2/14 ~3:30 pm with our CaSSIS geometry pipeline, testing the jigsaw observations mode fix and the label updates in tgocassisrdrgen. This error does not happen with either 5.0.2 or 6.0, which we use regularly with CaSSIS processing (in tandem to get around the jigsaw bug).

**Comment by volcanopele** [comment]
Tested spiceinit with a CTX image, J18_051828_191_XI_19N081W, with a very simple spiceinit command:

`spiceinit from=J18_051828_191_XI_19N081W.cal.cub`

and I got the same error as before.

**Comment by AustinSanders** [comment]
I'm having trouble reproducing this behavior on 7.0.0_RC1 (macOS BigSur 11.6.4).

<img width="1076" alt="Screen Shot 2022-02-18 at 12 51 40 PM" src="https://user-images.githubusercontent.com/3751180/154752128-a207da6a-b032-4921-8e7a-40a8a6aed097.png">

@volcanopele Can you verify that the process included in the screenshot accurately mirrors the process that led to the error you're experiencing?

**Comment by volcanopele** [comment]
For just my quick and easy test with a CTX image, those were the commands I used. The kernels used looks the same except for a slightly different SPK was used for MRO.

I'm not sure of the exact operating system of the system the CaSSIS pipeline runs on but it uses the Linux x86_64 version of ISIS 7.0. My tests with a CTX image were run on my office computer (same backend data files, but using macOS 64-bit version of ISIS 7.0 on macOS 11.5.2).

I'm going to install the release candidate on my personal system (macOS 12.2.1) so I can see if it might be an issue with the PIRL ISIS data area or with the installation on our systems. I will probably run that test on a CaSSIS image since I already have the TGO kernel files installed.

**Comment by volcanopele** [comment]
I'm working on trying to replicate this issue on my local system this morning. One quick question, the readme uses the following command for creating the ISIS environment:

`conda create -n isis python=3.6`

This has always worked fine for install ISIS but it is clear that 7.0.0 requires 3.7 or greater (due to protopuf from what I could tell). I requiring a specific version of Python still required or can we drop the version limiter when creating the environment.

With my current testing run on my MBP, I bumped it up to 3.7 (exact version is now 3.7.12). On the PIRL systems, it is version 3.9.10.

**Comment by volcanopele** [comment]
I re-re-installed ISIS 7.0.0 with python 3.9.10. I also found that I was missing a dependency, as I didn't have XQuartz on my local system. That caused issues with ISIS finding libGL.1.dylib. that's now resolved.

I found that I couldn't replicate the Spiceinit issue on my local system. Spiceinit ran fine with both the default shape model for Mars, $base/dems/molaMarsPlanetaryRadius0005.cub, and the DEM used by HiRISE.

So that still leaves the error we're getting at PIRL. One clue might be the difference in the version of Python? "python --version" reports 3.9.10 on my system (and when I previously installed ISIS with python 3.7, I got 3.7.12). Doing the same thing with the ISIS installation on PIRL, I get Python 3.7.6.
